README.md

```diff
@@ -1,4 +1,5 @@
-# Early Stopping Simulation
+# Early Stopping Simulation [![Python Versions](https://img.shields.io/badge/python-3.10-blue)]()
```

Comment:
edit the `python sample_run.py` line in README to point to the correct location.